NOTEA/Models/FileNameModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace NOTEA.Models
+{
+    public class FileNameModel
+    {
+        public string Name { get; set; }
+        public FileNameModel(string name)
+        { Name = name; }
+        public FileNameModel() 
+        { Name = ""; }
+    }
+}
```

Comment:
Mes siunčiam kur nors į puslapį šitą model? Nes jei ne, tai nežinau ar čia neturi būti paprastaa klasė? Bet čia ne esmė